app/modules/generateWallet.js

```diff
@@ -151,6 +151,13 @@ export const recoverWallet = (wallet: Object): Promise<*> =>
         reject(Error('No accounts found in recovery file.'))
       }
 
+      // check if wallet label already exists
+      const duplicateLabels = intersectionBy(data.accounts, accounts, 'label')
+
+      if (duplicateLabels.length) {
```

Comment:
@ranbena I think we should be explicit about number comparison... how about `duplicateLabels.length === 0`?

Reply:
Np, but why tho? It's readable and fail-safe.

Reply:
It's a fetish of mine.. writing code as minimal as possible.

Especially this one 🐒
```
if (~arr.indexOf('foo')) {
  ...
}
```

Reply:
or this one: `if (!0) { ... }` 😄

Reply:
Anywho, I updated the PR.